### PR TITLE
remove some useless configurations

### PR DESCRIPTION
**Pull request by rleungx**
Signed-off-by: Ryan Leung <rleungx@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

This PR removes some useless configurations of PD. 

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@rleungx Please involve technical reviews and fix the CircleCI failure. Thanks~

File: pd-configuration-file.md [hunk 1]
```diff
@@ -71,23 +71,6 @@ This document only describes parameters that are not included in command-line pa
 + Default value: `"pd-cluster"`
 + If multiple clusters that have nodes with same configurations are deployed successively, you must specify different tokens to isolate different cluster nodes.
 
-### `lease`
```
Comment (disksing) [inline pd-configuration-file.md:74]: why is this useless?

Comment (rleungx) [reply pd-configuration-file.md:74]: It is barely changed by users directly, so I think we can hide it.

Comment (TomShawn) [reply pd-configuration-file.md:74]: @disksing @rleungx Any update?

Comment (rleungx) [reply pd-configuration-file.md:74]: Ok, I am going to leave it for now.

**Review by nolouch — APPROVED** [review]
lgtm

**Review by disksing — APPROVED** [review]

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ You can adjust the PD scheduling parameters using [pd-ctl](/pd-control.md). Note
     >
     > The value of this parameter should be less than that of `region-schedule-limit`. Otherwise, the normal Region scheduling among TiKV nodes is affected.
 
-- [`store-balance-rate`](/pd-configuration-file.md#store-balance-rate): limits the rate at which Regions of each TiKV/TiFlash store are scheduled. Note that this parameter takes effect only when the stores have newly joined the cluster. If you want to change the setting for existing stores, use the following command.
+- `store-balance-rate`: limits the rate at which Regions of each TiKV/TiFlash store are scheduled. Note that this parameter takes effect only when the stores have newly joined the cluster. If you want to change the setting for existing stores, use the following command.
```
Comment (TomShawn) [inline tiflash/tiflash-configuration.md:21]: 如果这个配置项都在配置文件中删除了，这里是否也可以删除了呢？

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**